developer/debugging/vm-interface.md

```diff
@@ -44,6 +44,7 @@ QubesDB
 -   `/qubes-netvm-network` - network address (only when VM serves as network backend - ProxyVM and NetVM); can be also calculated from qubes-netvm-gateway and qubes-netvm-netmask
 -   `/qubes-netvm-primary-dns` - same as `qubes-primary-dns` in connected VMs (only when VM serves as network backend - ProxyVM and NetVM); traffic sent to this IP on port 53 should be redirected to primary DNS server
 -   `/qubes-netvm-secondary-dns` - same as `qubes-secondary-dns` in connected VMs (only when VM serves as network backend - ProxyVM and NetVM); traffic sent to this IP on port 53 should be redirected to secondary DNS server
+-   `/guivm-windows-prefix` - Prefix replacing `Dom0` from `[Dom0]` in window titles when GuiVM is not `dom0`. By default, window managers set `dom0` for every windows which are not created from ia `gui-daemon`. Every windows of a GuiVM itself are local and no `gui-daemon` is running for them.
```

Comment:
I think this can be simplified: title prefix for any window not originating from another qube. This means windows of applications running in GuiVM itself.